Mathlib/Geometry/Manifold/Instances/Real.lean

```diff
@@ -382,7 +384,7 @@ lemma IccRightChart_extend_top :
   norm_num [IccRightChart, modelWithCornersEuclideanHalfSpace_zero]
   congr
 
-lemma IccRightChart_extend_right_mem_frontier :
+lemma IccRightChart_extend_top_mem_frontier :
```

Comment:
This was merged today: if this PR gets reviewed quickly, I think omitting a deprecation is fine.

Reply:
Too bad it didn't get approved quickly :)

Reply:
Indeed; I've added a deprecation alias now.